src/main/java/seedu/address/logic/parser/AddressBookParser.java

```diff
@@ -46,60 +46,60 @@ public Command parseCommand(String userInput) throws ParseException {
+        case AddCommand.COMMAND_WORD:
+            return new AddCommandParser().parse(arguments);
 
-            case AddCommand.COMMAND_ALIAS:
-                return new AddCommandParser().parse(arguments);
+        case AddCommand.COMMAND_ALIAS:
+            return new AddCommandParser().parse(arguments);
 
-            case EditCommand.COMMAND_WORD:
-                return new EditCommandParser().parse(arguments);
+        case EditCommand.COMMAND_WORD:
+            return new EditCommandParser().parse(arguments);
 
-            case SelectCommand.COMMAND_WORD:
-                return new SelectCommandParser().parse(arguments);
+        case SelectCommand.COMMAND_WORD:
+            return new SelectCommandParser().parse(arguments);
 
-            case DeleteCommand.COMMAND_WORD:
-                return new DeleteCommandParser().parse(arguments);
+        case DeleteCommand.COMMAND_WORD:
+            return new DeleteCommandParser().parse(arguments);
 
-            case DeleteCommand.COMMAND_ALIAS:
-                return new DeleteCommandParser().parse(arguments);
+        case DeleteCommand.COMMAND_ALIAS:
+            return new DeleteCommandParser().parse(arguments);
 
-            case ClearCommand.COMMAND_WORD:
-                return new ClearCommand();
+        case ClearCommand.COMMAND_WORD:
+            return new ClearCommand();
 
-            case ClearCommand.COMMAND_ALIAS:
-                return new ClearCommand();
+        case ClearCommand.COMMAND_ALIAS:
+            return new ClearCommand();
 
-            case FindCommand.COMMAND_WORD:
-                return new FindCommandParser().parse(arguments);
+        case FindCommand.COMMAND_WORD:
+            return new FindCommandParser().parse(arguments);
 
-            case FindCommand.COMMAND_ALIAS:
-                return new FindCommandParser().parse(arguments);
+        case FindCommand.COMMAND_ALIAS:
+            return new FindCommandParser().parse(arguments);
 
-            case ListCommand.COMMAND_WORD:
-                return new ListCommand();
+        case ListCommand.COMMAND_WORD:
+            return new ListCommand();
 
-            case ListCommand.COMMAND_ALIAS:
-                return new ListCommand();
+        case ListCommand.COMMAND_ALIAS:
+            return new ListCommand();
 
-            case HistoryCommand.COMMAND_WORD:
-                return new HistoryCommand();
+        case HistoryCommand.COMMAND_WORD:
+            return new HistoryCommand();
 
-            case HistoryCommand.COMMAND_ALIAS:
-                return new HistoryCommand();
+        case HistoryCommand.COMMAND_ALIAS:
+            return new HistoryCommand();
 
-            case ExitCommand.COMMAND_WORD:
-                return new ExitCommand();
+        case ExitCommand.COMMAND_WORD:
+            return new ExitCommand();
 
-            case ExitCommand.COMMAND_ALIAS:
-                return new ExitCommand();
+        case ExitCommand.COMMAND_ALIAS:
```

Comment:
Why not make it cascade instead of code duplication? Just like here: https://github.com/se-edu/addressbook-level4/pull/590/files